vectorlm/trainer.py

```diff
@@ -59,12 +51,14 @@ class Trainer:
 
     """
 
+    peft_method: str | None = None
```

Comment:
Small nitpick but can we move this into an initialization declaration please? Just to keep style consistent.

Reply:
Sounds good- this actually might make a difference- on the hindsight, future torch versions might create issues when modifying class variables, which are shared across all instances of Trainer (not on other ranks, but might be the case in the future.)